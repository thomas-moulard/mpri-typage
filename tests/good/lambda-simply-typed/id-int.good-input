val id (x : int) = x
