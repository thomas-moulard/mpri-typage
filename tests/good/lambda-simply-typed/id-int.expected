id = fun (x : int).x

id : (int -> int)
